src/blockchains/zksync/protocols/macaronswap.ts

```diff
@@ -0,0 +1,45 @@
+import { Transaction } from '../types.ts';
+import { Protocol } from '../../types.ts';
+import { countTransactionPeriods } from '../utils.ts';
+
+const addresses: string[] = [
```

Comment:
You should lower the case of theses addresses otherwise the transactions will not match